src/components/charts/ohlcChart.tsx

```diff
@@ -40,18 +41,19 @@ const OHLCChart = ({ asset1Token, asset2Token }: OHLCChartProps) => {
     }
     // Get current block height from `/api/blocks/1`
     const getData = async () => {
-      const startBlock = await fetch("/api/blocks/1")
-        .then((res) => res.json())
-        .then((data) => {
-          const currentBlock = data[0].height
-          console.log(currentBlock)
-          const startBlock = currentBlock - Math.trunc(daysLookback * 24 * 60 * 60 / blockTimeSeconds);
-          console.log("Start block: ", startBlock);
-          return startBlock
+      const startBlock = await fetch('/api/blocks/1')
+        .then(res => res.json())
+        .then(data => {
+          const currentBlock = data[0].height;
+          console.log(currentBlock);
+          const startBlock =
+            currentBlock - Math.trunc((daysLookback * 24 * 60 * 60) / blockTimeSeconds);
+          console.log('Start block: ', startBlock);
+          return startBlock;
```

Comment:
_(not for later)_
Would be nice to stick with async/await or then()'s and not mix.